MANIFEST.in

```diff
@@ -1,8 +1,9 @@
-include *.in
-include *.py
+prune .github
 include *.md
-include LICENSE
 include tox.ini
-include pyproject.toml
-prune .travis
-prune debian
+
+# Make sure typing markers are included.
+recursive-include src py.typed
+
+# Include tests in the source distribution
+recursive-include tests *.py
```

Comment:
Do we need *.py here?

Reply:
Possibly not - this bit is copied from matrix-common's manifest.